CHANGES.rst

```diff
@@ -13,6 +13,18 @@ Changes
   The meta data us useful, for example, for selecting tests with the
   nose attribute mechanism.
 
+- Added ``doctestcase.doctestfiles``
+
+  - Define multiple doctest files at once.
+
+  - Automatically assign test class members.  So rather than::
+
+      test_foo = doctestcase.doctestfile('foo.txt')
+
+    You can use::
+
+      doctestcase.doctestfiles('foo.txt')
```

Comment:
My initial reaction is: ugh.  Magic.  It's not obvious that calling something in class scope defines new members.

I'd probably be happier with a class decorator, or a factory function (hey, isn't that just `doctest.DocFileCase(..., setUp=..., tearDown=...)`)